web/sites/default/config/user.role.administrator.yml

```diff
@@ -3,6 +3,7 @@ langcode: en
 status: true
```

Comment:
The changes in this file can be discarded, since administrator automatically has all permissions. (for some reason the field_permissions module adds a bunch of unnecessary perms here)